formats/json-jackson/src/main/java/io/cloudevents/jackson/JsonFormat.java

```diff
@@ -44,7 +44,7 @@ public final class JsonFormat implements EventFormat {
     /**
      * Content type associated with the JSON event format
      */
-    public static final String CONTENT_TYPE = "application/cloudevents+json";
+    public static final String CONTENT_TYPE = ContentType.JSON.value();
```

Comment:
You've probably already spotted [_this build failure_](https://github.com/cloudevents/sdk-java/actions/runs/4599824636/jobs/8137958743#step:4:7108) @pierDipi.

The problem is the same for both the two source files the Maven build reported. It looks like the Java™ SE 8 implementation of the `javax.ws.rs.Consumes` annotation only accepts strings that are "_constant values_".

Rolling back line `47` in `JsonFormat` (_and the corresponding one in the other two formats_) to the original hard coded value will fix that compilation error.

I'm open to alternative suggestions otherwise.

TIA.